redesign.user.js

```diff
@@ -797,14 +797,69 @@ const createSidebar = (id, position, icon, callback) => {
         document.getElementById('page')?.before(sidebar);
 
         // append the toggle button
-        document
-            .querySelector('#page .drawer-toggles')
-            ?.append(toggleBtnWrapper);
+        const togglesDiv =
+            document.querySelector(
+                `.${PREFIX(`drawer-toggles-${position}`)}`
+            ) ||
+            (() => {
+                const togglesDiv = document.createElement('div');
+                togglesDiv.classList.add(PREFIX(`drawer-toggles-${position}`));
+                document
+                    .querySelectorAll(`.drawer-${position}-toggle`)
+                    .forEach(toggler => {
+                        togglesDiv.append(toggler);
+                    });
+                document
+                    .querySelector('#page .drawer-toggles')
+                    .append(togglesDiv);
+                return togglesDiv;
+            })();
+        togglesDiv.append(toggleBtnWrapper);
 
         callback(content, header);
     });
 };
 
+GM_addStyle(`
+    /* Sidebars */
+    .drawer-toggles {
+        position: fixed;
+        top: var(--navbar-height);
+        left: 0;
+        width: 100vw;
+        margin-top: 0.7rem;
+        margin-bottom: 0.7rem;
+        z-index: 100;
+    }
+    .better-moodle-drawer-toggles-right,
+    .better-moodle-drawer-toggles-left {
+        display: flex;
+        flex-direction: column;
+        position: fixed;
+        gap: 0.7rem;
+    }
+    .better-moodle-drawer-toggles-right {
+        right: 0;
+    }
+    .better-moodle-drawer-toggles-left {
+        left: 0;
+    }
+    .drawer-toggler {
+        position: initial !important;
+    }
+    .drawer-toggles .drawer-toggler .btn .icon.fa-fw {
+        width: 16px; /* Reset to .icon default */
+    }
+    @media (max-width: 767.98px) {
+        .better-moodle-drawer-toggles-right,
+        .better-moodle-drawer-toggles-left {
```

Comment:
We should the `PREFIX` function here to avoid hardcoding.

Reply:
This is resolved with https://github.com/YorikHansen/better-moodle/commit/de4cedef5cfe570e4618cb42745c5cd088006dd9